bazarr/embedded_subs_reader.py

```diff
@@ -21,6 +21,7 @@ def list_languages(self, file):
             data = api.know(file)
 
             traditional_chinese = ["cht", "tc", "traditional", "zht", "hant", "big5", u"繁", u"雙語"]
+            brazilian_portuguese = ["pt-br", "pob", "pb", "Brazilian", "Brasil"]
```

Comment:
Those string should all be lowercase as you are comparing to `detected_language['name'].lower()`